LayoutTests/svg/zoom/page/text-with-non-scaling-stroke.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
```

Comment:
svg/zoom/page/text-with-non-scaling-stroke.html fails when LBSE is activated.
All other tests work fine both in the legacy engine and LBSE.

Will investigate the remaining LBSE error tomorrow morning.